recipe/meta.yaml

```diff
@@ -8,7 +8,7 @@ package:
 source:
    fn: {{ name }}-{{ version }}.tar.gz
    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-   sha256: 34a9f35f95c441e7b38209775d6e0337f9a3759f3565f6c5798f19618527c76f
+   sha256: 7afc856f74fa7006a289fd10fa840e1eebd8bbff6bffb69c26c54a0512ea8cf8
 
 build:
   noarch: python
```

Comment:
Please change the build number to `0`, because it's a build of a new version

Reply:
The build number was set to `0` since it is a new version

https://github.com/AnacondaRecipes/filelock-feedstock/blob/update_to_3.3.2/recipe/meta.yaml#L15